src/main/java/uk/gov/hmcts/cmc/claimstore/controllers/ClaimController.java

```diff
@@ -133,6 +133,17 @@ public CreatePaymentResponse resumePayment(
         return claimService.resumePayment(authorisation, claimData);
     }
 
+    @PutMapping(value = "/{submitterId}/create-citizen-claim", consumes = MediaType.APPLICATION_JSON_UTF8_VALUE)
+    @ApiOperation("Creates a citizen claim")
+    public Claim createClaim(
+        @Valid @NotNull @RequestBody ClaimData claimData,
+        @RequestHeader(HttpHeaders.AUTHORIZATION) String authorisation
```

Comment:
`front-end` passes list of `features` which are enabled on the claim, and we save as part of that claim, we have those missing here.

Reply:
👍